CMakeLists.txt

```diff
@@ -237,6 +246,8 @@ if (${PROJECT_NAME}_ENABLE_OPENMP)
   endif ()
 endif (${PROJECT_NAME}_ENABLE_OPENMP)
 
+find_package(MPI 3.0 REQUIRED COMPONENTS CXX)
```

Comment:
This isn't necessary; it's taken care of in FindAndVerifyMPI.cmake

Reply:
Done.